src/Task.ts

```diff
@@ -41,6 +43,32 @@ export function handlePriorityFeeLimitSet(event: PriorityFeeLimitSet): void {
   task.save()
 }
 
+export function handleTokensAcceptanceTypeSet(event: TokensAcceptanceTypeSet): void {
+  const task = Task.load(event.address.toHexString())
+  if (task == null) return log.warning('Missing task entity {}', [event.address.toHexString()])
+
+  const acceptanceType = parseAcceptanceType(event.params.acceptanceType)
+  const acceptanceListId = task.id
+  const acceptanceList = loadOrCreateTokenAcceptanceList(acceptanceListId)
+  acceptanceList.tokensAcceptanceType = acceptanceType
+  acceptanceList.save()
+}
+
+export function handleTokensAcceptanceListSet(event: TokensAcceptanceListSet): void {
+  const task = Task.load(event.address.toHexString())
+  if (task == null) return log.warning('Missing task entity {}', [event.address.toHexString()])
+
+  const tokensAcceptanceListId = task.id
+  const tokensAcceptanceList = loadOrCreateTokenAcceptanceList(tokensAcceptanceListId)
+  const tokens = tokensAcceptanceList.tokens
+  const token = event.params.token.toHexString()
```

Comment:
Simply to make sure we track that token information if it wasn't tracked before:

```suggestion
  const token = loadOrCreateERC20(event.params.token).id
```